aws_lambda_powertools/logging/formatter.py

```diff
@@ -144,6 +144,8 @@ def __init__(
 
         if self.utc:
             self.converter = time.gmtime
+        else:
+            self.converter = time.localtime
```

Comment:
What you found here to simplify the solution was incredible! You always pay close attention to details. :clap: